gradle_plugin/src/main/resources/template/testSpec.yml

```diff
@@ -0,0 +1,51 @@
+# See detailed explanation for parameters in https://github.com/microsoft/HydraLab/blob/main/gradle_plugin/src/main/resources/template/gradle.properties
```

Comment:
If these files are not used in runtime, and they are acting as doc/samples, we may move them in the module root, and we can reference them in MD file or wiki.